fenix/app/src/main/java/org/mozilla/fenix/onboarding/interactor/OnboardingInteractor.kt

```diff
@@ -38,4 +52,28 @@ class DefaultOnboardingInteractor(
     override fun onReadPrivacyNoticeClicked() {
         controller.handleReadPrivacyNoticeClicked()
     }
+
+    override fun onMenuItemTapped(item: SearchSelectorMenu.Item) {
+        searchSelectorController.handleMenuItemTapped(item)
+    }
+
+    override fun onLearnMoreClicked() {
+        privateBrowsingController.handleLearnMoreClicked()
+    }
+
+    override fun onPrivateModeButtonClicked(newMode: BrowsingMode, userHasBeenOnboarded: Boolean) {
+        privateBrowsingController.handlePrivateModeButtonClicked(newMode, userHasBeenOnboarded)
+    }
+
+    override fun onNavigateSearch() {
+        toolbarController.handleNavigateSearch()
+    }
+
+    override fun onPaste(clipboardText: String) {
+        toolbarController.handlePaste(clipboardText)
+    }
+
+    override fun onPasteAndGo(clipboardText: String) {
+        toolbarController.handlePasteAndGo(clipboardText)
+    }
```

Comment:
are these exact functions also in session control interactor? I'm assuming these are necessary because the app is fully accessible from within Onboarding and when it hasn't been dismissed yet.